project/web/balancer.sls

```diff
@@ -170,6 +170,7 @@ install_letsencrypt:
   git.latest:
     - name: https://github.com/letsencrypt/letsencrypt/
     - target: {{ letsencrypt_dir }}
+    - force_checkout: True
```

Comment:
Here's the output from epicallies... It's failing on the pull, so I think we'll need reset.

```
[staging.epicallies.com] out: ----------
[staging.epicallies.com] out:           ID: install_letsencrypt
[staging.epicallies.com] out:     Function: git.latest
[staging.epicallies.com] out:         Name: https://github.com/letsencrypt/letsencrypt/
[staging.epicallies.com] out:       Result: False
[staging.epicallies.com] out:      Comment: Command 'git pull ' failed. Stderr: 'error: Your local changes to the following files would be overwritten by merge:\n\tletsencrypt-auto\nPlease, commit your changes or stash them before you can merge.\nAborting'
[staging.epicallies.com] out:      Started: 18:02:50.402623
[staging.epicallies.com] out:     Duration: 1114.521 ms
[staging.epicallies.com] out:      Changes:   
[staging.epicallies.com] out: ----------
```

Hopefully, that will be enough. I know @dpoirier rewrote the [margarita checkout using manual commands](https://github.com/caktus/django-project-template/blob/master/conf/salt/margarita.sls) because the salt version of those commands weren't reliable before.